help.go

```diff
@@ -39,7 +39,10 @@ func deprecatedCommandHelpFunc(commandNames []string, f cli.HelpFunc) cli.HelpFu
 		sort.Strings(keys)
 
 		for _, key := range keys {
-			commandFunc := commands[key]
+			commandFunc, ok := commands[key]
+			if !ok {
+				continue
+			}
```

Comment:
Why did this change?

Reply:
Needed for the `--help` command to work for some reason